src/expressServer.js

```diff
@@ -128,6 +129,28 @@ app.use('/api/v1/wishlist', wishlistRoutes /* #swagger.tags = ['Wishlist'] */);
 app.use('/admin', require('./routes/adminRoutes'));
 app.use(express.static(__dirname + '/public'));
 
+// Add this route to serve the order notifications page
+app.get('/order-notifications', (req, res) => {
+  res.render('orderNotifications'); // Render the EJS template
+});
+
+app.use(
+  '/toastify',
+  expressStaticGzip('./node_modules/toastify-js/src', { enableBrotli: true })
+);
+
+app.get('/toastify/toastify.js', (req, res) => {
+  res.setHeader('content-type', 'application/javascript');
+  const jsFile = readFileSync('./node_modules/toastify-js/src/toastify.js');
+  res.send(jsFile);
+});
+
+app.get('/toastify/toastify.css', (req, res) => {
+  res.setHeader('content-type', 'text/css');
+  const cssFile = readFileSync('./node_modules/toastify-js/src/toastify.css');
+  res.send(cssFile);
+});
+
```

Comment:
If `expressStaticGzip` works the way that I think it does, then it will already be serving these files from the appropriate node\_modules forlder with the appropriate `Content-Type` headers. So lines 142 to 152 can be removed

Reply:
Since I no longer have an error in the console, do you think it's better for me to remove this code or express-static-gzip? They both works well